Local Setup Issue: Command failed: cd packages/amplication-server && npm run start:db

### Overview

When locally setting up amplication.

`npm run setup:dev` was failing with the following error. 
```
info: Starting step 6/7
✔ Finished running docker compose up

info: Starting step 7/7
✖ Command failed: cd packages/amplication-server && npm run start:db
npm ERR! Missing script: "start:db"
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     npm start # Start a package
npm ERR!     npm run start:debug # run the "start:debug" package script
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2022-06-12T16_05_24_485Z-debug.log
```

### Symptoms

This is happening at `db seeding` step (which is last) of setup process (`npm run setup:dev`).

### Cause

The Reason for this error is that ` start:db` command is not present in `packages/amplication-serve` package. Instead , it exist in `packages/amplication-prisma-db ` 


### Resolution

The Solution for this issue is update `script/setup.ts`

![image](https://user-images.githubusercontent.com/37527349/173244463-48e7b408-de55-4840-a3eb-35ed085fbd29.png)


### Tags

setup

### Additional Information

Setup docs for https://github.com/amplication/amplication/blob/master/packages/amplication-server/README.md also needs to be updated

![image](https://user-images.githubusercontent.com/37527349/173244523-234c1e3b-0bf2-4d87-b91e-32b65cd3ef2c.png)


### Related Discord Threads

https://discord.com/channels/757179260417867879/985534859063742494

### Are you willing to submit PR?

Yes I am willing to submit a PR!

--- a/packages/amplication-server/README.md
+++ b/packages/amplication-server/README.md
@@ -50,7 +50,7 @@ You can also use a more manual step-by-step approach to set up Amplication serve
   ```
   npm run docker
   ```
-- Update application database (execute in server directory "packages/amplication-server")
+- Update application database (execute in server directory "packages/amplication-prisma-db")
 
   ```
   npm run start:db

--- a/script/setup.ts
+++ b/script/setup.ts
@@ -99,7 +99,7 @@ const docker: Task[] = [
 ];
 const dockerInit: Task[] = [
   {
-    command: "cd packages/amplication-server && npm run start:db",
+    command: "cd packages/amplication-prism-db && npm run start:db",
     label: "db seeding",
   },
 ];

